bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableParser.scala

```diff
@@ -140,24 +178,25 @@ trait VoTableParser {
       k.sequenceU
     }
 
-    def combineWithErrors(m: List[Magnitude], e: Map[MagnitudeBand, Double]) = m.map(i => i.copy(error = e.get(i.band)))
+    def combineWithErrorsAndFilter(m: List[Magnitude], e: Map[MagnitudeBand, Double]): List[Magnitude] =
+      m.map(i => i.copy(error = e.get(i.band))).filterNot(magnitudesFilter.nonValidMagnitude)
```

Comment:
Perhaps reversing this to `.filter(magnitudesFilter.validMagnitude)` would be a bit better?